test/alternator/test_export.py

```
@@ -0,0 +1,1191 @@
+    table_arn2 = get_table_arn(test_table_s_2)
+    s3 = make_s3_client(dynamodb)
+    token = str(uuid.uuid4())
+
+    with new_s3_bucket(s3) as bucket:
+        client1.export_table_to_point_in_time(
+            TableArn=table_arn1,
+            S3Bucket=bucket,
+            ExportFormat='DYNAMODB_JSON',
+            ClientToken=token,
+        )
+        with pytest.raises(ClientError, match='ExportConflictException.*Duplicate request detected'):
```

Ruff (0.16.3)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)